app/config/sculpin_kernel.yml

```diff
@@ -26,3 +26,8 @@ sculpin_content_types:
             - category
             - authors
         permalink: pretty
+    8:
```

Comment:
Why a new content type? Do you plan on doing something with `_views/8.html`? Control a common footer or CTA? include a partial on all Drupal 8 pages with it's own set of links?